code/game/objects/items/ego_weapons/he.dm

```diff
@@ -125,12 +127,30 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+	crit_multiplier = 3	//Give a better crit chance.
 
 //ATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATATAT
 /obj/item/ego_weapon/paw/melee_attack_chain(mob/user, atom/target, params)
 	..()
 	hitsound = "sound/weapons/punch[pick(1,2,3,4)].ogg"
 
+/obj/item/ego_weapon/paw/CritEffect(mob/living/target, mob/living/carbon/human/user)
+	for(var/turf/T in orange(1, user))
+		new /obj/effect/temp_visual/smash_effect(T)
+
+	for(var/mob/living/L in range(1, user))
+		var/aoe = force
+		var/userjust = (get_modified_attribute_level(user, JUSTICE_ATTRIBUTE))
+		var/justicemod = 1 + userjust/100
+		aoe*=force_multiplier
+		aoe*=justicemod
+		if(L == user || ishuman(L))
+			continue
+		L.apply_damage(aoe, RED_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
```

Comment:
Damage here should be calculated outside of the loop (We aren't expecting nor really caring if the user's stats change somehow in the middle of this instantaneous attack). This could also thereby utilize HitInTurf to prevent crits from hitting allies seemingly randomly.